homeassistant/components/bmw_connected_drive/binary_sensor.py

```diff
@@ -225,7 +225,7 @@ def __init__(
         self.entity_description = description
         self._unit_system = unit_system
 
-        self._attr_name = f"{vehicle.name} {description.key}"
+        self._attr_name = description.name
```

Comment:
I think that's the default, and you can drop the line.
Same applies on the other platforms...